ADR/ADR-82-crashbot.md

```diff
@@ -55,61 +55,40 @@ stateDiagram-v2
 
 ### Endpoints
 
-- `POST /create` Creates a new incident
+- `GET /list` Returns
   ```
-  body
-  { 
-    severity: string,
-    report_date: string,
-    description: string,
-    point: string,
-    contact: string
-  }
-  ```
-
-- `PUT /update/:incident_id` Updates an existing incident
-  ```
-  body
-  { 
-    severity: string, (optional)
-    report_date: string, (optional)
-    description: string, (optional)
-    point: string, (optional)
-    contact: string, (optional)
-    resolution_eta: string, (optional)
-    rca_link: string (optional)
-  }
-  ```
-- `PUT /close/:incident_id` Closes an existing incident
-  ```
-  body
-  { 
-    resolution_eta: string,
-    rca_link: string (optional)
-  }
-  ```
-
-- `GET /list` Returns a 
-  ```
-  response
   {
     open: [
       {
+        id: number,
+        update_number: number,
+        modified_at: string,
+        modified_by: string
+        reported_at: string,
+        closed_at: string,
+        status: string,
         severity: string,
-        report_date: string,
+        title: string,
         description: string,
         point: string,
-        contact: string
+        contact: string,
+        rca_link: string
       }
     ],
     closed: [
       {
+        id: number,
+        update_number: number,
+        modified_at: string,
+        modified_by: string
```

Comment:
```suggestion
```

Should we trait the person behind the report as anonymous?

Reply:
Every other incident manager keep track of who is making the changes. It is useful whenever you need to ask about the incident or its changes

Reply:
But that information is protected, not publicly available

Reply:
Great point. We could store it for future usage but not expose it then, what do you think?